desktop/core/src/desktop/templates/global_js_constants.mako

```diff
@@ -138,6 +138,7 @@
   window.SHOW_NOTEBOOKS = '${ SHOW_NOTEBOOKS.get() }' === 'True'
   window.SHOW_UPLOAD_BUTTON = '${ hasattr(SHOW_UPLOAD_BUTTON, 'get') and SHOW_UPLOAD_BUTTON.get() }' === 'True'
   window.UPLOAD_CHUNK_SIZE = '${ UPLOAD_CHUNK_SIZE.get() }';
+  window.UPLOAD_SIZE_LIMIT = ${ UPLOAD_SIZE_LIMIT.get() };
```

Comment:
If we don't wanna change the python default I think we should follow the example of the DOWNLOAD_ROW_LIMIT and write something like:

`window.UPLOAD_SIZE_LIMIT = ${ UPLOAD_SIZE_LIMIT.get() if UPLOAD_SIZE_LIMIT.get() >= 0 else 'undefined' };`

Reply:
Yes, we can do this. 
Thanks will make changes accordingly.

Reply:
Done